### PR TITLE
Pointers in C

### PR DESCRIPTION
**Pull request by nimra200**
I created the article "Pointers in C". The article is in the file index.md .
My article explains what pointers are and gives code examples that explain how to declare and use them. I also discuss how strings and arrays relate to pointers, and clarify misconceptions that beginners may have. 
There are no extra images added.

### CODE REVIEW

**Comment by nadivgold** [comment]
@nimra200 Do you want to include an image of yourself for the author card at the bottom?

File: articles/pointers-in-c/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+#include <stdio.h> 
+int main(){
+    int i = 5;
+    int *p;
+    p = &i; /* p points to i */
+    printf(" The value of i is %d \n", *p); // Dereferencing p gives i
+    printf(" The memory address of i is %p\n ", p); 
+    return 0;
+}
+```
```
Comment (nadivgold) [inline articles/pointers-in-c/index.md:41]: Can you add the output of this program for those who don't run the code?

File: articles/pointers-in-c/index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+``` c 
+char myString [3]; 
+myString[0] = 'P'; 
+myString[1] = 'i';
+myString[2] = '\0';
+*myString = 'H'; // myString is a pointer to the character 'H'
+```
+### Putting Your Knowledge to the Use
+
+You've learnt that pointers are memory addresses. Arrays use pointers, and we can dereference pointers to access elements of an array. Strings are arrays too, so we can manipulate them with pointers as well.  I recommend that you read *C Programming: A Modern Approach* by K. N. King for further reference. The book has exercises and programs that you can test out too.
```
Comment (nadivgold) [inline articles/pointers-in-c/index.md:86]: You've learnt that pointers -> You've learned that pointers

Comment (nadivgold) [reply articles/pointers-in-c/index.md:86]: The book has exercises and programs that you can test out too.  -> The book also has exercises and programs that you can test out.

File: articles/pointers-in-c/index.md (hunk 1 above)
Comment (nimra200) [reply articles/pointers-in-c/index.md:41]: I've added the program output as per your suggestion on line 42.

File: articles/pointers-in-c/index.md (hunk 2 above)
Comment (nimra200) [reply articles/pointers-in-c/index.md]: Ok, I've committed these changes now. Thank you for your suggestion!

**Comment by nimra200** [comment]
I've added a headshot of myself and added my linkedin to my author bio. Please let me know if I did it incorrectly.

**Comment by nadivgold** [comment]
> I've added a headshot of myself and added my linkedin to my author bio. Please let me know if I did it incorrectly.

yep looks good to me

**Comment by hectorkambow** [comment]
Plagiarism check done

**Comment by hectorkambow** [comment]
👋  Good afternoon Molly @ninjaginja - wanted to ask if you could quickly give this article a final tech review. 
The first article by the contributor - Nadiv did a great job adding the front matter - and hero image - and did a preliminary/tech review.
Look quite good - had a few code examples.
Thank you

**Pull request merged**